File: v2.x/Examples/SciChartDemo/SciChartDemo/ExampleViews/CreateRealtimeCharts/RealTimeGhostedTracesPanel.m
//
//  RealTimeGhostedTracesPanel.m
//  SciChartDemo
//
//  Created by Yaroslav Pelyukh on 4/30/17.
//  Copyright © 2017 ABT. All rights reserved.
//

#import "RealTimeGhostedTracesPanel.h"

@implementation RealTimeGhostedTracesPanel

- (IBAction)sliderChangedValue:(UISlider *)sender {
    self.msTextLabel.text = [NSString stringWithFormat:@"%.0f ms", sender.value];
    
    if (self.speedChanged) {
        self.speedChanged(sender);
    }
}

@end

File: v2.x/Examples/SciChartSwiftDemo/SciChartSwiftDemo/ImportantFiles/RealTimeGhostedTracesPanel.swift
//
//  RealTimeGhostedTracesPanel.swift
//  SciChartSwiftDemo
//
//  Created by Yaroslav Pelyukh on 5/1/17.
//  Copyright © 2017 SciChart Ltd. All rights reserved.
//

import UIKit
typealias RealTimeGhostedTracesPanelCallback = (_ sender:UISlider) -> Void

class RealTimeGhostedTracesPanel: UIView{
    
    @IBOutlet weak var msText: UILabel!
    var callBack: RealTimeGhostedTracesPanelCallback?
    
    @IBAction func speedChangedPresesd(_ sender: UISlider) {
        msText.text = String.init(format: "%.0f ms", sender.value)
        
        if let call = callBack{
            call(sender)
        }
    }
}
